flexget/plugins/filter/exists_movie.py

```diff
@@ -23,17 +20,40 @@ class FilterExistsMovie(object):
       exists_movie: /storage/movies/
     """
 
-    schema = one_or_more({'type': 'string', 'format': 'path'})
+    schema = {
+        'anyOf': [
+            one_or_more({'type': 'string', 'format': 'path'}),
+            {
+                'type': 'object',
+                'properties': {
+                    'path': one_or_more({'type': 'string', 'format': 'path'}),
+                    'allow_different_qualities': {'enum': ['better', True, False], 'default': False},
+                    'type': {'enum': ['files', 'dirs'], 'default': 'dirs'},
+                    'imdb_lookup': {'type': 'boolean', 'default': False}
```

Comment:
Perhaps change this to "lookup: imdb" so it will be forward compatible if we later add support for other lookup facilities?